app/commands/source/version_changelog_command.rb

```diff
@@ -0,0 +1,141 @@
+# frozen_string_literal: true
+
+module Source
+  class VersionChangelogCommand < ApplicationCommand
+    usage do
+      no_command
+    end
+
+    option :version do
+      long "--version string"
+      desc "The version of the full_names.yml file to compare"
+    end
+
+    def execute
+      version = params[:version] || latest_version
+      frame("Processing categories for version #{version}") do
+        ensure_exports_directory
+        delete_previous_exports(version)
+        process_category_tree
+        compare_and_export(version)
+      end
+    end
+
+    private
+
+    def latest_version
```

Comment:
This file has version in it... not sure if that will help you?
https://github.com/Shopify/product-taxonomy/blob/main/VERSION